tests/admin_scripts/tests.py

```diff
@@ -2363,7 +2373,6 @@ def test_program_name_from_argv(self):
 
 @override_settings(ROOT_URLCONF="admin_scripts.urls")
 class StartProject(LiveServerTestCase, AdminScriptTestCase):
-
```

Comment:
Please remove this (and other) unrelated change(s).

Reply:
Thanks for the feedback, I have made the changes.